src/customize-pop/default-applications.md

```diff
@@ -0,0 +1,83 @@
+# Default Applications
+
+Options to set the default web browser, email client, calendar application, music player, video player, and photo viewer are located in `Settings` ➞ `Default Applications`. This menu allows setting a single application as the default handler for a larger number of similar file types.
+
+![Default Applications](/images/application-settings/default-applications.png)
+
+Designate an application as the default by selecting it from the dropdown menu. If no appropriate application is installed to handle a media type, the dropdown selection will be grayed out. <!--link to installing applications section when complete-->
+
+![Select Default](/images/application-settings/select-default.png)
+
+## Set Default Applications Using the File Manager
+
+Launch the file manager and navigate to the file you want to open. Right click the file and select `Properties`.
+
+![File Properties](/images/application-settings/file-properties.png)
+
+Click the `Open With` tab and choose the appropriate program from the list.
+
+![Open With](/images/application-settings/open-with.png)
+
+## Set Default Applications Using the Terminal
+
+### Modify the Mimeapps.list File
+
+The `mimeapps.list` file contains entries associating applications with specific links and MIME file types. Application names listed in this file will be appended with `.desktop`.
+
+1. Determine your application's .desktop name using this command to list all .desktop applications:
+
+    ```bash
+    ls /usr/share/applications
```

Comment:
This will show most system-wide apps installed, but it won't show user-local apps; apps can be placed in any of the directories you see in the output of `echo $XDG_DATA_DIRS`. For example, Flatpak apps are installed to `~/.local/share/flatpak/exports/share/applications/` by default.

Reply:
I'm going to add this comment as a note in the documentation.